custom_components/rivian/const.py

```diff
@@ -577,6 +583,55 @@
             icon="mdi:car-battery",
             entity_category=EntityCategory.DIAGNOSTIC,
         ),
+        RivianSensorEntityDescription(
+            key="limited_acceleration_cold",
+            field="limitedAccelCold",
+            name="Limited Acceleration (Cold)",
+            icon="mdi:snowflake-thermometer",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        RivianSensorEntityDescription(
+            key="limited_regen_braking_cold",
+            field="limitedRegenCold",
+            name="Limited Regenerative Braking (Cold)",
+            icon="mdi:snowflake-thermometer",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        RivianSensorEntityDescription(
+            key="bluetooth_front_fascia_hardware_failure_status",
+            field="btmFfHardwareFailureStatus",
+            name="Bluetooth Module Failure Status Front Fascia",
```

Comment:
Do we want to go one more step and do "Fascia Front/Rear" and "Door Front Left/Right"? I'm fine either way since I think the more important grouping is BT module failure status.

Reply:
changed this to match other sensors with "Front/Rear Right/Left" as with tires and enclosures